### PR TITLE
Create AppShell layout, Reuse Footer, StaticAppBar #2158

### LINKED ISSUE #2158: Create AppShell layout
**Actual Behaviour**

Not present

**Expected Behaviour**

Implement AppShell layout consisting of Header and Footer, as said by @akshatnitd 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2158

Changes: Reuse Footer, StaticAppBar

Demo Link: https://pr-2159-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -93,6 +99,7 @@ class App extends Component {
             message={snackBarMessage}
             onClose={actions.closeSnackBar}
           />
+          {pathname !== '/' && <StaticAppBar />}
```
Comment (akshatnitd) [inline src/App.js]: Keep this logic inside StaticAppBar, would be more cleaner.

File: src/App.js [hunk 2]
```diff
@@ -157,6 +164,7 @@ class App extends Component {
             <ProtectedRoute exact path="/settings" component={Settings} />
             <Route exact path="/*:path(error-404|)" component={NotFound} />
           </Switch>
+          {pathname !== '/' || (skillListRegex.test(pathname) && <Footer />)}
```
Comment (akshatnitd) [inline src/App.js]: Same here.

File: src/App.js (hunk 1 above)
Comment (fragm3) [reply src/App.js]: @akshatnitd Yes, but instead of rendering the component and returning it null, this might be a better way.

Comment (akshatnitd) [reply src/App.js]: I agree. But, the difference is negligible. So, let's go for cleaner code 💯

Comment (akshatnitd) [reply src/App.js]: Now, it looks fine. But, once there would be multiple logic. We will need to anyways do it.

Comment (fragm3) [reply src/App.js]: @akshatnitd I tried what you are trying to say, after implmenting it, it is problematic keeping the current code in mind. If I put the logic in StaticAppBar, I am getting this error:

`Uncaught TypeError: Cannot read property 'offsetHeight' of undefined
    at StaticAppBar.componentDidMount (StaticAppBar.react.js:217)`

Screenshot 2019-06-03 at 7 44 31 PM

Also, if we don't return anything, we will have to nest it into React.Fragment which is not desirable either.

**Comment by fragm3** [comment]
@akshatnitd I have used another approach to render the elements, please review.

**Pull request merged**